app/src/Homepage/Homepage.tsx

```diff
@@ -53,11 +53,11 @@ export const RatingBox: React.FC<{label: string, rating: string}> = ({ label, ra
             {label}
         </div>
     );
-  };
```

Comment:
It seems that the changes here are the exact same? Try to revert the changes if they are exactly the same?

Reply:
These changed removed inconsistent/unneeded spacing. The end line of RatingBox was indented for no reason. The last RatingDescProps was indented for no reason.